build.cmd

```diff
@@ -1,2 +1,3 @@
-@call "%~dp0run.cmd" build-managed %*
-@exit /b %ERRORLEVEL%
\ No newline at end of file
+@echo off
+powershell -ExecutionPolicy ByPass -NoProfile %~dp0eng\Build.ps1 -restore -build -warnaserror:0 -nodereuse:0 %*
+echo Build.cmd ErrorLevel=%ERRORLEVEL%
```

Comment:
should we exit with the ERRORLEVEL instead of just echoing it?

Reply:
Yes we should exit with it. I added the echo in corefx to help with diagnosing exit codes across scripts but we I think we can leave it in.